tests/basic_integration/test.sh

```
@@ -18,19 +18,22 @@ export RUST_LOG=info
 RPC_URL="ws://localhost:8545"
 
 PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+CIPHERNODE_SECRET="We are the music makers and we are the dreamers of the dreams."
 
 # These contracts are based on the deterministic order of hardhat deploy
 # We _may_ wish to get these off the hardhat environment somehow?
 ENCLAVE_CONTRACT="0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
```

Shellcheck

[warning] ENCLAVE_CONTRACT appears unused. Verify use (or export if used externally).

(SC2034)